includes/core/classes/class-settings.php

```diff
@@ -778,4 +779,25 @@ public function select_menu( $submenu ): string {
 		return (string) $submenu;
 	}
 
+	/**
+	 *  Display notice if users can't register
+	 *
+	 * @return void
+	 */
+	public function check_users_can_register() {
+		if ( filter_var( get_option( 'users_can_register' ), FILTER_VALIDATE_BOOLEAN ) ) {
+			return;
+		}
+		?>
+			<div class="notice notice-warning is-dismissible">
+				<p>
+					<?php
+						/* translators: %s: search term */
+						echo sprintf( esc_html__( 'For GatherPress to work, you will want to allow users to <a href=%s>register</a>.', 'gatherpress' ), esc_url( admin_url( 'options-general.php#users_can_register' ) ) );
```

Comment:
To ensure GatherPress functions optimally, we recommend enabling user registration. You can do so by <a href=%s>enabling user registration here</a>.